spartan/aztec-network/templates/boot-node.yaml

```diff
@@ -128,12 +128,12 @@ spec:
                   fieldPath: status.podIP
             - name: AZTEC_PORT
               value: "{{ .Values.bootNode.service.nodePort }}"
+            - name: DEBUG
```

Comment:
is debug still needed here

Reply:
Yeah, in some places we use DEBUG for libp2p debugging

Reply:
Was going to ask if we can still specify `DEBUG=libp2p:*` and get ouytput from libp2p? I assume this means yes?

Reply:
Yep! I had actually removed `DEBUG` from the prover-node and validator as well, I just added them back so we can debug libp2p there as well.